packages/events/ReactControlledComponent.js

```diff
@@ -35,11 +37,13 @@ function restoreStateOfTarget(target) {
   restoreImpl(internalInstance.stateNode, internalInstance.type, props);
 }
 
-export function setRestoreImplementation(impl) {
+export function setRestoreImplementation(
+  impl: (Element, string, Object) => void,
```

Comment:
maybe let's name the arguments?

Reply:
Do tell me if anything else is required.